tests/checkpointing/test_legacy_checkpoints.py

```diff
@@ -60,6 +60,7 @@
         "1.2.5",
         "1.2.6",
         "1.2.7",
+        "1.2.8",
```

Comment:
do you know why this is failing the test?

Reply:
Because 1.2.8 hasn't been released yet